app/script/auth/route.js

```diff
@@ -19,13 +19,18 @@
 
 export default {
   CHOOSE_HANDLE: '/choosehandle',
+  CLIENTS: '/clients',
   CONVERSATION_JOIN: '/join-conversation',
+  CONVERSATION_JOIN_INVALID: '/join-conversation-invalid',
   CREATE_ACCOUNT: '/createaccount',
   CREATE_TEAM: '/createteam',
   CREATE_TEAM_ACCOUNT: '/createteamaccount',
+  HISTORY_INFO: '/historyinfo',
   INDEX: '/',
   INITIAL_INVITE: '/teaminvite',
   INVITE: '/invite',
+  LOGIN: '/login',
+  PASSWORD_RESET: '/forgot',
```

Comment:
Is this really an internal route or rather an external path we set so that it does not really belong here?